front_end/ndb/NdbMain.js

```diff
@@ -208,6 +238,26 @@ Ndb.NodeProcessManager = class extends Common.Object {
         info.id, userFriendlyName(info), SDK.Target.Type.Node,
         this._targetManager.targetById(info.ppid) || this._targetManager.mainTarget(), undefined, false, connection);
     target[NdbSdk.connectionSymbol] = connection;
+
+    try {
+      // this doesnt work
+      // target.runtimeAgent().invoke_evaluate({
+      //   expression: `
+      //     const zlib = require('http');
+      //     console.log('foo');
+      //   `
+      // });
+      //
+      // but this does
+      // target.runtimeAgent().invoke_evaluate({
+      //   expression: `
+      //     console.log('foo');
```

Comment:
this doesn't have `import` and it worked. I can see the console in the `ndb` log, but the context was `VMxx`. So I'm not sure we could monkey patch the http/https request this way. Please correct me if I'm wrong.